BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c

```diff
@@ -306,6 +319,23 @@ FrameBufferWrite (
     return 0;
   }
 
+  // Handle clear screen
+  if ((NumberOfBytes == 4) && (!CompareMem (ANSI_ESCAPE_SEQ_CLEAR_SCREEN, Buffer, 4))) {
```

Comment:
the return of CompareMem is INTN. should compare with 0 by EDK2 coding style.

Reply:
Will change it.